lib/policy_templates/security/removeHeaderAuthorization.js

```diff
@@ -0,0 +1,48 @@
+/**
+Copyright 2022 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+var builder = require('xmlbuilder')
+var random = require('../../util/random.js')
+
+module.exports = {
+  removeHeaderAuthorizationTemplate: removeHeaderAuthorizationTemplate,
+  removeHeaderAuthorizationGenTemplate: removeHeaderAuthorizationGenTemplate
+}
+
+function removeHeaderAuthorizationTemplate (options) {
+  var aysnc = options.async || 'false'
+  var continueOnError = options.continueOnError || 'false'
+  var enabled = options.enabled || 'true'
```

Comment:
'enabled' is assigned a value but never used.

Reply:
Pls run `npm run test` to find the errors. The test doesnt run due to the above mentioned

Reply:
Sorry @ssvaidyanathan, i have pushed the fixes and also the name on the new test was wrong!